index.html

```diff
@@ -1,123 +1,191 @@
+            sign-ups required. has been the industry's standard
+          </p>
+          <ul class="mg-1 mg-inline flex tag tag-project">
+            <li><a href="#">html</a></li>
+            <li><a href="#"> bootstrap </a></li>
+            <li><a href="#"> Ruby </a></li>
+          </ul>
+          <button class="btn btn-project" type="button">See Project</button>
         </div>
+      </div>
     </section>
-</body>
-</html>
\ No newline at end of file
+    <section class="bg-dark container">
+      <div class="about">
+        <h2 class="text-primary-1 ff-crete">About Me</h2>
+        <p class="mg-1 ff-inter text-light">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <button class="btn" type="button">Get my Resume</button>
+      </div>
+      <div class="flex skill">
+        <h3 class="text-light ff-inai">
+          <span class="skill-icon"></span> Language
+        </h3>
+        <ul class="center ff-inter">
+          <li class="text-light mg-1"><a href="#">Javascript</a></li>
+          <li class="text-light mg-1"><a href="#"> Ruby </a></li>
+          <li class="text-light mg-1"><a href="#"> HTML </a></li>
+          <li class="text-light mg-1"><a href="#"> CSS </a></li>
+        </ul>
+        <h3 class="text-light ff-inai">
+          <span class="skill-icon skill-icon-kite"></span> Frameworks
+        </h3>
+        <ul class="center ff-inter">
+          <li class="text-light mg-1"><a href="#">Bootstrap</a></li>
+          <li class="text-light mg-1"><a href="#"> Ruby on Rails</a></li>
+          <li class="text-light mg-1"><a href="#">RSPec</a></li>
+          <li class="text-light mg-1"><a href="#">CapyBara</a></li>
+          <li class="text-light mg-1"><a href="#"> Selenium </a></li>
+        </ul>
+        <h3 class="text-light ff-inai">
+          <span class="skill-icon skill-icon-circle"></span> Skills
+        </h3>
+        <ul class="center ff-inter">
+          <li class="text-light mg-1"><a href="#">Codekit</a></li>
+          <li class="text-light mg-1">
+            <a href="https://github.com/t-bello7"> Github</a>
+          </li>
+          <li class="text-light mg-1"><a href="#"> Codepen </a></li>
+          <li class="text-light mg-1"><a href="#">Gitlab</a></li>
+          <li class="text-light mg-1"><a href="#"> Terminal </a></li>
+        </ul>
+      </div>
+    </section>
+    <footer class="container"> 
+      <div class="contact-form mg-8 ">
+        <h2 class="center ff-inter">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form class="mg-2 flex contact" action="https://formspree.io/f/mwkywooe">
```

Comment:
- [ ] Kindly add the `post` attribute to your form, currently, data is not sent to the form spree server.